### DISCUSSION TITLE
Strange rendering problems (styles partially missing?)

### DISCUSSION THREAD

**Discussion by datokrat** [athensresearch/athens#1232] [Archive]
Did anyone else get this scary look recently? It started to occur today in development builds (old and new ones; recompiling helped, sometimes). This is my productive Athens version, not updated today. It even happens when I download the AppImage again, and it's irrelevant whether I choose BETA-80 or BETA-82. Switching the database also doesn't help. So I believe the problem might be on my side and not in some recent commits and this probably does not qualify as a bug.

![image](https://user-images.githubusercontent.com/6992158/119386541-faf7f500-bcc7-11eb-8a7f-d09ea92397e5.png)

I really cannot explain this. I recently noticed Leiningen complaining about some NPM version problems:
```
NPM version conflict for "react" in deps.cljs (will use A)
A: "17.0.1" from jar:file:/home/paul/.m2/repository/day8/re-frame/re-frame-10x/1.0.2/re-frame-10x-1.0.2.jar!/deps.cljs
B: "16.13.1" from jar:file:/home/paul/.m2/repository/devcards/devcards/0.2.7/devcards-0.2.7.jar!/deps.cljs
NPM version conflict for "react-dom" in deps.cljs (will use A)
A: "17.0.1" from jar:file:/home/paul/.m2/repository/day8/re-frame/re-frame-10x/1.0.2/re-frame-10x-1.0.2.jar!/deps.cljs
B: "16.13.1" from jar:file:/home/paul/.m2/repository/devcards/devcards/0.2.7/devcards-0.2.7.jar!/deps.cljs
NPM dependency "react" has installed version "16.9.0"
"17.0.1" was required by jar:file:/home/paul/.m2/repository/day8/re-frame/re-frame-10x/1.0.2/re-frame-10x-1.0.2.jar!/deps.cljs
NPM dependency "react-dom" has installed version "16.9.0"
"17.0.1" was required by jar:file:/home/paul/.m2/repository/day8/re-frame/re-frame-10x/1.0.2/re-frame-10x-1.0.2.jar!/deps.cljs
NPM dependency "highlight.js" has installed version "9.15.10"
"10.7.1" was required by jar:file:/home/paul/.m2/repository/superstructor/re-highlight/0.0.1/re-highlight-0.0.1.jar!/deps.cljs
```

However, these only affect my dev builds and the problem also occurs in the stable BETA-82 AppImage, battle-proven the last days.

Does anyone have an idea what might be the problem?

**Comment by datokrat** [comment]
It seems to have to do with local storage (after renaming the local storage folder in `.config/Athens/Local Storage`, it works again). So maybe this *does* qualify as a bug. Let's see whether I can find the problem.

**Reply by datokrat** [reply]
Problem solved! It's good to note that you can edit the local storage either using `cmd-alt-i` or `cmd-shift-i`, depending on the system. I deleted the entry `stylefy_cache_styles` and now it works again.

I'm not sure what we can learn from this for the future -- maybe a "panic button" that you can press if something does not work for flushing caches, showing hints etc.?

**Reply by jefftangx** [reply]
Yeah this is from stylefy cache. Whenever I see weird CSS it's my default response to delete the stylefy key. It seems like pretty tribal knowledge to know. I wonder if there is a better solution here for devs new to Athens/Stylefy, cc @shanberg
